cucumber/authenticators_gcp/features/authn_gcp_basic_host.feature

```diff
@@ -0,0 +1,41 @@
+Feature: GCP Authenticator - Hosts can authenticate with GCP authenticator
+
+  In this feature we define an Azure authenticator in policy and perform authentication
+  with Conjur, using a host with subscription-id & resource-group annotations.
+  In successful scenarios we will also define a variable and permit the host to
+  execute it, to verify not only that the host can authenticate with the Azure
+  Authenticator, but that it can retrieve a secret using the Conjur access token.
+
+  Background:
+    Given a policy:
+    """
+    - !policy
+      id: conjur/authn-gcp
+      body:
+      - !webservice
+
+      - !group apps
+
+      - !permit
+        role: !group apps
+        privilege: [ read, authenticate ]
+        resource: !webservice
+    """
+    And I am the super-user
+    And I have host "test-app"
+    And I set a GCP annotation with name: "authn-gcp/service-account-id" and value: "gcp_service_account_id" to host: "test-app"
+    And I grant group "conjur/authn-gcp/apps" to host "test-app"
+
+  Scenario: Hosts can authenticate with GCP authenticator and fetch secret
+    Given I have a "variable" resource called "test-variable"
+    And I permit host "test-app" to "execute" it
+    And I add the secret value "test-secret" to the resource "cucumber:variable:test-variable"
+    And I obtain a GCE identity token in "full" format and audience claim value: "conjur/cucumber/test-app"
+    And I save my place in the audit log file
+    When I authenticate with gcp-authn using GCP identity token as host "test-app"
+    Then host "test-app" has been authorized by Conjur
+    And I successfully GET "/secrets/cucumber/variable/test-variable" with authorized user
```

Comment:
And I can successfully...

Reply:
That is an existing step in `request_steps.rb`

Reply:
yes but you can add a `can` before `GET`. when we use this step in `WHEN` we use `successfully GET`. when we use it in `THEN` we use `can successfully`. It reads better that way.

Reply:
it's xor `( (?:can|successfully))?`

Reply:
then use `I can GET` for THEN